### DISCUSSION TITLE
Is it possiable to mergo `range` into `select`?

### DISCUSSION THREAD

**Discussion by gipsyh** [nushell/nushell#5341] [Ideas]
can we merge `range` command into `select`?
for example: `ls | select 0..2 name`

**Comment by kubouch** [comment]
It should be possible to add ranges to `select`, that sounds nice. We would need to discuss removing the `range` command because it's a breaking change but it seems quite redundant if `select` supports ranges.

Can you open a feature request issue for it? And if you feel like working on it, we would accept a PR.

**Reply by gipsyh** [reply]
> Can you open a feature request issue for it? And if you feel like working on it, we would accept a PR.

sure, #5344

**Comment by fdncred** [comment]
I'm not a fan of removing the `range` command. However, I would like to see this range syntax used in several commands like `select`, `first`, `last`, `take`, `skip`, and maybe others. Some of those commands may already support a range.
